### ISSUE TITLE
Prints for the stairs and entrance

### ISSUE THREAD

**Issue by iteles** [dwyl/home#188]
I'd like to swap out the A2 prints on the stairs and in the entrance for a variety of interesting prints, a mix of quotes and others.

@nelsonic Proposed, please let me know if you'd prefer others:
https://www.etsy.com/listing/1121726362/educational-birds-poster-common-birds
![image](https://user-images.githubusercontent.com/4185328/150637424-a5d321fb-b5ca-47c7-8fdf-13ff987dfee5.png)

https://www.etsy.com/listing/916172174/educational-bugs-poster-insects-print
![image](https://user-images.githubusercontent.com/4185328/150637538-5b7a4625-e4bd-4724-b381-397b102c0a53.png)

And then these quotes:

EITHER
![image](https://user-images.githubusercontent.com/4185328/150637682-224b9776-c145-43bc-b588-d84eaf3df1bd.png)
Learn from yesterday, live for today, hope for tomorrow

OR
![image](https://user-images.githubusercontent.com/4185328/150637707-c89eb333-0290-4550-9563-e8896ee0f289.png)

Imagination is more important than knowledge. For while knowledge defines all we currently know and understand, imagination points to all we might yet discover and create.

OR any other quote you like.

On _this_ background:
https://www.etsy.com/listing/938954754/custom-quote-printable-with-sun-line-art
![image](https://user-images.githubusercontent.com/4185328/150637661-56a02cbc-5d0e-4f1b-a78b-b34c745bbb7b.png)

AND

“It is what you read when you don't have to that determines what you will be when you can't help it.” - Oscar Wilde

On _this_ background (which I have already bought for something else):
https://www.etsy.com/listing/585313794/greenery-quote-print-custom-quote-print
![image](https://user-images.githubusercontent.com/4185328/150637835-e1ac68d9-b20f-49a4-9c09-1b95b1d7dae1.png)

**Comment by nelsonic** [comment]
@iteles _really_ like the idea of the birds. But would strongly prefer _local_ ones over North American ones.
This could be a sub/evening-task for **FJ** who no doubt has *hundreds* of great photos of the PT/Iberian birds and could send us some high-res pics to print. Sure it wouldn't be a _single_ poster, but it would also be a lot more _meaningful_. 

The Bugs one is great. except for the misclassification of the [**`Coccinella transversalis`**](https://en.wikipedia.org/wiki/Coccinella_transversalis) as a "***bug***" 🐞 ... it's a ***`beetle`***!! 🙄  

Totally down for the quotes. Hence crowdsourcing them: https://github.com/dwyl/quotes
Kinda prefer to "print" (etch) them on our laser cutter on 3-5mm plywood ... we could start our _own_ Etsy Store! 😉 

![image](https://user-images.githubusercontent.com/194400/150638726-d330013f-b923-4aad-a0e1-304a042ba4c9.png)

This is "C-Grade" (construction) eucalyptus plywood. But we could easily get some A-Grade Baltic Birch https://github.com/dwyl/home/issues/123 and it would look _siiiiick_ ... 

Our Laser can _easily_ etch a photo or drawing e.g: https://youtu.be/gMS7IZZ6oVI

![image](https://user-images.githubusercontent.com/194400/150638811-7ad21963-9886-43f3-a1c2-60b6c2ac6c49.png)

> **Note**: definitely not saying we should make something tacky like this! I'm sure the person who ordered it _loved_ it tho. 😜 

Boggles my mind what people will pay $99 + shipping for! 🤯 
https://laserperfectgifts.com/engraved-wood-baby-dedication-plaque/
![image](https://user-images.githubusercontent.com/194400/150639191-ca5e8873-59e6-4e2a-8276-5c971a7b622f.png)

But back to the quotes ... 
I would put a "scribble" of Einstein above the quote, e.g: https://www.google.com/search?q=einstein+scribble&tbm=isch
https://static.displate.com/857x1200/displate/2019-10-12/a5e6449dcd9e07cf102b384818366e25_01ebd9cc92241e0aef832a8ab107738d.jpg

![image](https://user-images.githubusercontent.com/194400/150639255-9baf0200-1295-4b19-be16-f1acbf613d4e.png)

Though I'm sure @Stephanymtr could help you find a much more attractive one with like 10 mins of searching. 💡

**Comment by iteles** [comment]
I agree about the birds and I have already asked him for this but it will take a WHILE. In the meantime I would like something up on the walls.

How about flowers:
https://www.etsy.com/listing/964655303/flowers-downloadable-prints-watercolor
![image](https://user-images.githubusercontent.com/4185328/150645710-023f7c6b-3769-4d18-a6cd-f771e20075c7.png)

Culinary herbs:
https://www.etsy.com/listing/976422305/herbs-print-kitchen-wall-decor
![image](https://user-images.githubusercontent.com/4185328/150646175-8400c6df-7ddf-44e3-9491-df4f8cebd809.png)


Mushrooms
![image](https://user-images.githubusercontent.com/4185328/150645752-c0c6882f-ed8f-4ca3-879c-d9116b807adc.png)


Or colours
![image](https://user-images.githubusercontent.com/4185328/150645762-4aaf0cdc-1543-4f7c-8073-1e284ad2585d.png)

**Comment by iteles** [comment]
@Stephanymtr 
Could you please also put your amazing Canva skills into making a simple poster with this quote:

"It is what you read when you don't have to that determines what you will be when you can't help it."
- Charles Francis Potter

![—Pngtree—single line drawing of stack_5045408](https://user-images.githubusercontent.com/4185328/154681417-784508ac-8dab-4b5d-b054-ca15f95a6ec6.png)


Email I need to send these to later:
geral@maisdecopias.com

**Comment by Stephanymtr** [comment]
Albert Einstein drafts: Here are some drafts I was able to put together. I can always change colors, fonts etc. This document was saved as a "PDF para impressao"  which should have high quality :)

[Albert Einstein Prints - First draft.pdf](https://github.com/dwyl/home/files/8097120/Albert.Einstein.Prints.-.First.draft.pdf)

**Comment by Stephanymtr** [comment]
Charles Francis Potter draft

[Charles Francis Potter draft.pdf](https://github.com/dwyl/home/files/8097359/Charles.Francis.Potter.draft.pdf)

**Comment by iteles** [comment]
Amazing thank you!

Final roundup of posters that went for printing:
- Mushrooms
- Birds
- Colours
- Flowers
- Bugs
- Charles Potter quote
- Albert Einstein quote

If we don't like any of them printed out we can always replace 👍
